apps/ledger-live-desktop/static/i18n/en/app.json

```
@@ -3525,6 +3525,19 @@
       },
       "providerTitle": "Earn with {{provider}}",
       "providerLink": "Learn more about {{provider}}"
+    },
+    "stakeV2": {
+      "kiln": {
+        "title": "Kiln validator",
+        "description": "Got more than 32 ETH? Stake on dedicated validator. Earn rewards daily. No lock-up period.",
+        "tag": "Requires 32 ETH",
+        "supportLink": "Learn more about Kiln validator"
+      },
+      "kiln_pooling": {
+        "title": "Kiln pooling",
+        "description": "Participate in a pool and stake any ETH amount. No lock-up period. No minimum.",
+        "supportLink": "Learn more about Kiln pooling"
+      }
```

I think we can avoid v2 here too.
(If you do not like the structure feel free to change it)

A little of context, Lido is a pooling provider which we are in negotiation... I would leave this prepared in case we need a quick provider change using only a FF.

```suggestion
    "stake": {
      "title": "Earn Ethereum",
      "lido": {
        "title": "Lido",
        "description": "Participate in a pool and stake any ETH amount. No lock-up period. No minimum.",
        "supportLink": "Learn more about Lido"
      },
      "kiln": {
        "title": "Kiln validator",
        "description": "Got more than 32 ETH? Stake on dedicated validator. Earn rewards daily. No lock-up period.",
        "tag": "Requires 32 ETH",
        "supportLink": "Learn more about Kiln validator"
      },
      "kiln_pooling": {
        "title": "Kiln pooling",
        "description": "Participate in a pool and stake any ETH amount. No lock-up period. No minimum.",
        "supportLink": "Learn more about Kiln pooling"
      }
```

---

Yes, makes sense!